components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -4666,7 +4666,7 @@ public boolean updateApplicationOwner(String userName, Application application)
         try {
             connection = APIMgtDBUtil.getConnection();
             String driverName = connection.getMetaData().getDriverName();
-            if (driverName.contains("MS SQL") || driverName.contains("Microsoft") || driverName.contains("Oracle")) {
+            if (driverName.contains("Oracle")) {
                 offset = start + offset;
```

Comment:
Seems like this overriding a previous fix.

https://github.com/wso2/carbon-apimgt/pull/9324/files#diff-ac8fc1b7b2d10089084f9f842ad03ac096b64f8c555798aa18a926e0ec663a4cR4644-R4645

Reply:
from the above fix, the string "oracle" is added to the checklist, where from this one the other ones are removed except "oracle"

Reply:
seems like this is ok since the check is added for oracle.

Reply:
Ah.... OK